x-pack/plugins/secops/public/components/timeline/body/renderers/row_renderer.ts

```diff
@@ -4,9 +4,17 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import styled from 'styled-components';
+
 import { Ecs } from '../../../../graphql/types';
 
+export const RowRendererContainer = styled.div<{ width: number }>`
+  width: ${({ width }) => `${width}px`};
```

Comment:
thanks for adding this to ensure the custom row renderer's width is based on the width of the visible timeline! 😄